### ISSUE TITLE
Accessibility: invalid attribute aria-role

### ISSUE THREAD

**Issue by fitnycdigitalinitiatives** [omeka/omeka-s#2111]
Running an accessibility checker on our site and caught this on the annotations: application/view/common/resource-values.phtml

Should be 'role' rather than 'aria-role'

**Comment by zerocrates** [comment]
Thanks for the report, I've changed/fixed the role attributes there.
